File: abc/abc233/a.cpp
// https://atcoder.jp/contests/abc233/submissions/28114306

#include <bits/stdc++.h>
using namespace std;
int main() {
  int x, y, cnt = 0; cin >> x >> y;
  for(; x < y; x += 10) cnt++;
  cout << cnt;
}

File: abc/abc233/a.cs
// https://atcoder.jp/contests/abc233/submissions/46506090

using System;

public class A
{
  static void Main()
  {
    string[] t = Console.ReadLine().Split(" ");
    int x = int.Parse(t[0]);
    int y = int.Parse(t[1]);

    int ans = 0;
    while(x < y)
    {
      ans++;
      x += 10;
    }

    Console.Write(ans);
  }
}

